gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -319,6 +322,24 @@
   private static final Compute.AddressAggregatedListOption ADDRESS_AGGREGATED_LIST_FILTER =
       Compute.AddressAggregatedListOption.filter(ADDRESS_FILTER);
 
+  // Snapshot options
+  private static final Compute.SnapshotOption SNAPSHOT_OPTION_FIELDS =
```

Comment:
Nit: A lot of options are already imported; could do the same for these as well if you want.